ocs_ci/utility/tnf_hypervisor.py

```
@@ -0,0 +1,937 @@
+        logger.info(f"Waiting for SSH on {self.public_ip} (timeout {timeout}s)...")
+
+        for sample in TimeoutSampler(
+            timeout=timeout,
+            sleep=30,
+            func=self._try_ssh_connect,
+        ):
+            if sample:
+                logger.info("SSH connection established to hypervisor")
+                return
+
+    def _try_ssh_connect(self):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_try_ssh_connect`

Add return type annotation: `bool`

(ANN202)